applications/GeoMechanicsApplication/custom_processes/apply_excavation_process.hpp

```diff
@@ -11,118 +11,68 @@
 //                   Aron Noordam,
 //                   Vahid Galavi
 //
-//
-
-
-// System includes
-#include <cmath>
-#include <iostream>
-#include<string>
+#pragma once
 
 // Project includes
 #include "includes/model_part.h"
-#include "utilities/openmp_utils.h"
-#include "utilities/math_utils.h"
 #include "includes/element.h"
 
 // Application includes
-
-#if !defined(KRATOS_GEO_APPLY_EXCAVATION_PROCESS )
-#define  KRATOS_GEO_APPLY_EXCAVATION_PROCESS
-
 #include "includes/kratos_flags.h"
 #include "includes/kratos_parameters.h"
 #include "processes/process.h"
 
-#include "geo_mechanics_application_variables.h"
 
 namespace Kratos
 {
 
 class ApplyExcavationProcess : public Process
 {
   public:
-
-    typedef std::size_t IndexType;
-    typedef Table<double, double> TableType;
-
     KRATOS_CLASS_POINTER_DEFINITION(ApplyExcavationProcess);
 
-    //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-    /// Constructor
     ApplyExcavationProcess(ModelPart&  model_part,
                            Parameters& rParameters) : Process(Flags()), mrModelPart(model_part)
```

Comment:
```suggestion
                           Parameters rParameters) : Process(Flags()), mrModelPart(model_part)
```

due to internal management, `Parameters` should be passed by value

Reply:
Thanks for pointing that out. I didn't know that.